src/__fixtures__/upload-details-rejected-empty.js

```diff
@@ -0,0 +1,26 @@
+const uploadDetailsRejectedEmptyFixture = {
+  redirect:
+    'http://redirect.com/plants/add/status-poller?uploadId=58f10d6e-4b5e-4edc-8c80-5fda853bb6ac',
+  destinationBucket: 'cdp-example-node-frontend',
+  destinationPath: '/plants',
+  metadata: {
+    plantId: '94f8a562-630c-4569-b3a3-2503408c4129'
+  },
+  uploadId: '58f10d6e-4b5e-4edc-8c80-5fda853bb6ac',
+  uploadStatus: 'pending',
+  initiated: '2024-04-29T10:44:56.989Z',
+  fields: {
+    button: 'upload',
+    file: {
+      contentType: 'application/octet-stream',
+      hasError: true,
+      errorMessage: 'The selected file is empty',
+      fileStatus: 'pending',
```

Comment:
In this fixture  should the fileStatus be pending, not rejected?

Reply:
It may be the test flow, just checking

Reply:
I think the latest commit specifically sets this to rejected??

Reply:
removed - ta 👍